onnxruntime/test/contrib_ops/dynamic_slice_op_test.cc

```diff
@@ -8,37 +8,37 @@ namespace onnxruntime {
 namespace test {
 
 TEST(DynamicSliceTest, dynamic_slice_varied_types) {
-  OpTester test1 ("DynamicSlice", 1);
+  OpTester test1 ("DynamicSlice", 9);
```

Comment:
Need not be an opset 1 model. Model can be opset 9 and it will take the latest opset version of the operator from the def (which is 1)